src/Shared/E2ETesting/SeleniumStandaloneServer.cs

```diff
@@ -111,7 +111,7 @@ private static async Task InitializeInstance(ITestOutputHelper output)
         var psi = new ProcessStartInfo
         {
             FileName = "npm",
-            Arguments = $"run selenium-standalone start -- --config \"{seleniumConfigPath}\" {chromeDriverArg} -- -host {SeleniumHost} -port {port}",
+            Arguments = $"run selenium-standalone start -- --config \"{seleniumConfigPath}\" {chromeDriverArg} -- --host {SeleniumHost} --port {port}",
```

Comment:
Arguments for host and port require double hyphens

Reply:
This is interesting. How this used to work then in the past (or today)?

Reply:
It was using an older version